py3status/modules/keyboard_layout.py

```diff
@@ -3,9 +3,15 @@
 Display keyboard layout.
 
 Configuration parameters:
+    button_next: Button to click to cycle up through available layouts
+        (default 4)
+    button_prev: Button to click to cycle up through available layouts
+        (default 5)
+    button_select: Button to click to select current layout (default 1)
```

Comment:
We `cycle up` twice.
We should decide which to standardize `button_prev` or `button_previous` too.
`button_action` in favor of generic naming /  putting this on the top?

Reply:
thanks for the `up` spot.

yeah the button nameing could be better.  I need to get my remove all the button configs PR sorted then we will not have this problem.  We will still have a naming issue but it will be a different one.